themes/gatsby-theme-datocms-blog/gatsby-node.js

```diff
@@ -26,6 +26,12 @@ exports.createSchemaCustomization = async ({ actions }) => {
       url: String
     }
 
+    type DatoCmsBlogauthor implements Node & BlogAuthor {
+      id: ID!
+      name: String
+      avatar: Image
+    }
+
     type DatoCmsBlogpost implements Node & BlogPost {
       id: ID!
```

Comment:
Wouldn't we need to have a reference to the blog author in this `DatoCmsBlogpost` type?

Reply:
Yup, looks like I thought I'd finished this but missed DatoCMS